pkg/auth/authorizer/abac/abac.go

```diff
@@ -51,13 +60,74 @@ func (p policyLoadError) Error() string {
 
 type policyList []*api.Policy
 
+type ABACPolicy struct {
+	Auth atomic.Value
+}
+
+func New(path string) (*ABACPolicy, error) {
+
+	var plist policyList
+	abacPolicy := &ABACPolicy{}
+
+	plist, err, eof := newFromFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	if eof {
+		go wait.Forever(func() {
+			var lastModifiedTime *time.Time
+			for {
+				time.Sleep(filePollPeriod)
+
+				done := make(chan error)
+				go func() {
+					info, err := os.Stat(path)
+					if err != nil {
+						done <- err
+					} else {
+						modTime := info.ModTime()
+						if lastModifiedTime == nil || *lastModifiedTime == modTime {
```

Comment:
skipping on nil `lastModifiedTime` means an update immediately after starting the server, but before the first poll, would be ignored